src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -875,6 +875,8 @@ public PreprocessingSymbolInfo GetPreprocessingSymbolInfo(SyntaxNode nameSyntax)
 
         internal virtual Func<SyntaxNode, bool>? GetSyntaxNodesToAnalyzeFilter(SyntaxNode declaredNode, ISymbol declaredSymbol) => null;
 
+        internal abstract bool IsSynthesizedRecordConstructor(ISymbol symbol);
```

Comment:
>internal abstract bool IsSynthesizedRecordConstructor(ISymbol symbol); [](http://example.com/codeflow?start=8&length=70)

This change looks surprising to me. I felt that we could simply revert relevant changes made in [https://github.com/dotnet/roslyn/pull/46308](https://github.com/dotnet/roslyn/pull/46308).